src/main/kotlin/solve/importer/view/ControlPanel.kt

```diff
@@ -22,18 +25,20 @@ class ControlPanel : View() {
 
     private val importButtonModel = ButtonModel()
 
+    private val coroutineScope = CoroutineScope(Dispatchers.Main)
```

Comment:
what if we close window? will it be cancelled? why do we separate it in variable if never use anywhere?